packages/govuk-frontend/src/govuk/components/header/_index.scss

```diff
@@ -30,6 +30,16 @@
     color: $govuk-header-text;
   }
 
+  .govuk-header--with-js-navigation {
```

Comment:
Should these changes be scoped within `_govuk-rebrand`? Is there a risk of it having ramifications on non-rebrand headers?

Reply:
Hmm, good question - I'm trying to think of an adverse affect it might have on older headers - I guess some folks might be _relying_ on the early reflow?

In other words, I'm torn between "this is a decent fix for non-rebrand stuff" and "let's limit this scope as much as possible"

Reply:
Given our component is only meant to have a service name and/or the navigation after the logo, offering it to everyone rather than just for the rebrand feels pretty safe to me as we wouldn't be changing the behaviour for people using GOV.UK Frontend and its public options 😊